pom.xml

```diff
@@ -130,7 +120,7 @@
 		</repository>
 		<repository>
 			<id>org.geppetto-mvn-repo</id>
-            <url>https://raw.github.com/openworm/org.geppetto.maven/mvn-repo</url>
+            <url>https://raw.github.com/openworm/org.geppetto.maven/mvn-repo-nml1.5.2</url>
```

Comment:
@tarelli I've opened a PR for the 1.5.2 jars at https://github.com/openworm/org.geppetto.maven/pull/6

I've also got a list of things to change/tidy up after everything is merged to development here: https://github.com/OpenSourceBrain/geppetto-osb/issues/49